lib/internal/util/inspect.js

```diff
@@ -1294,6 +1294,11 @@ function formatError(err, constructor, tag, ctx, keys) {
     keys.push('cause');
   }
 
+  // Print errors aggregated into AggregateError
+  if (err.errors && (keys.length === 0 || !keys.includes('errors'))) {
```

Comment:
I don't think a truthy check is the most appropriate here
```suggestion
  if (errors in err && (keys.length === 0 || !keys.includes('errors'))) {
```
or
```suggestion
  if (ArrayIsArray(err.errors) && (keys.length === 0 || !keys.includes('errors'))) {
```